src/test/java/org/folio/circulation/domain/OverdueFineCalculatorServiceTest.java

```diff
@@ -211,19 +210,22 @@ public void shouldNotCreateFeeFineRecordWhenItemDoesNotExist()
     Loan loan = createLoan();
 
     when(overdueFinePolicyRepository.findOverdueFinePolicyForLoan(any()))
-      .thenReturn(CompletableFuture.completedFuture(Result.succeeded(loan)));
+      .thenReturn(completedFuture(succeeded(loan)));
     when(overduePeriodCalculatorService.getMinutes(any(), any()))
-      .thenReturn(CompletableFuture.completedFuture(Result.succeeded(5)));
+      .thenReturn(completedFuture(succeeded(5)));
```

Comment:
Ok, I got it. These values are hardcoded in all of the test where fine is NOT charged, so it doesn't matter what the calculator returns. Anyway, I would replace them with the proper value from the test params (`periodCalculatorResult`) just in case.

Reply:
done